src/model/ServerNode.ts

```diff
@@ -507,15 +522,24 @@ File extensions: ${formatOptionalString(sastConfig.fileExtension)}
                 }
             }
 
+            
+            var isPrivate : boolean;
+            if(!isProjPrivate) // Checking if project is private all its scans are by default private. 
+           //Hence no need of asking user another prompt to choose whether scan has to be private/public
+            {
             const isScanPrivate = await Utility.showPickString("Is scan private?", ['Yes', 'No']);
-            const isPrivate: boolean = Utility.modeIsEnabled(isScanPrivate);
```

Comment:
Can we declare isPrivate variable once. Since it is already defined above if can we remove from if block

Reply:
@prachipalaskar23  its the deleted line on which you have commented